src/borg/crypto/key.py

```diff
@@ -454,7 +498,41 @@ def encrypt_key_file(self, data, passphrase):
         )
         return msgpack.packb(enc_key.as_dict())
 
-    def _save(self, passphrase):
+    def encrypt_key_file_argon2(self, data, passphrase):
+        # https://www.rfc-editor.org/rfc/rfc9106.html#section-4-6.2
+        time_cost = 3
+        memory_cost = 2**16
+        parallelism = 4
+        type_ = 'id'
```

Comment:
for usage in `borg benchmark cpu`, it would be nice to have a constant with this, like we have for PBKDF2_ITERATIONS - so benchmark and real usage stay in sync.